test-fixtures/sort-relator-terms/f02/expectedResult.json

```diff
@@ -28,7 +35,7 @@
       {"code": "a", "value": "Relator term po. $j:ssä #2"},
       {"code": "j", "value": "säveltäjä."},
```

Comment:
Period in `säveltäjä.` is not correct, but should it be handled somewhere else?

Reply:
Good catch! Sort-relator-terms uses punctuation2 fixer for changed fields. Updated punc removal rules fot X11$e, and now this is handled properly.